app/views/layouts/application.html.erb

```diff
@@ -8,7 +8,11 @@
 </head>
 <body>
 
-<%= yield %>
+  <% flash.each do |name, msg| %>
+    <%= content_tag :div, msg, id: "flash_#{name}" %>
+  <% end %>
+
+  <%= yield %>
 
```

Comment:
I found this flash_mesages code while putting authy into a project and thought it looked pretty good.  Allows you to call <%= flash_messages %> in the application.html.erb.
https://github.com/TwilioDevEd/authy2fa-rails/blob/master/app/helpers/application_helper.rb